### ISSUE TITLE
.Net Core

### ISSUE THREAD

**Issue by lgoudriaan** [NightOwl888/BoboBrowse.Net#24]
Hi @NightOwl888,

Any plans on making an asp.net core release?

**Comment by NightOwl888** [comment]
Lucene 3.0.3 doesn't support .NET Core, so BoboBrowse 3.x isn't going to have support for it. However, I had updated to BoboBrowse 4.0.2 more than 6 months ago, but only recently has Lucene.Net 4.8 been put on NuGet.

So, here is a MyGet feed with the alpha of 4.0.2 with .NET Standard 1.5 support. Let me know if you run into any trouble with it.

https://www.myget.org/F/bobobrowse_net/api/v2 (Visual Studio 2012+)
https://www.myget.org/F/bobobrowse_net/api/v3/index.json (Visual Studio 2015+)

I will put it on NuGet once it has been established it is stable enough, the documentation, and the demo have been updated.
